auth/store.go

```diff
@@ -153,6 +156,9 @@ type AuthStore interface {
 
 	// Close does cleanup of AuthStore
 	Close() error
+
+	// AuthInfoFromCtx gets AuthInfo from gRPC's context
```

Comment:
In `AuthInfoFromCtx()`, gRPC's metadata is obtained by `metadata.FromContext()`. So I think the context cannot be said generic.